composer.json

```diff
@@ -18,9 +18,10 @@
         "monolog/monolog": "^1.2",
```

Comment:
Class `UserEntity` has deprecated method

Reply:
ignoring this one because we're not using `UserEntity` and its technically still working in php `7.4`